aws_lambda_builders/workflows/nodejs_npm_esbuild/esbuild.py

```diff
@@ -99,28 +99,7 @@ def run(self, args, cwd=None):
         return out.decode("utf8").strip()
 
 
-# The esbuild API flags are broken up into three forms (https://esbuild.github.io/api/):
-# Multi-word arguments are expected to be passed down using snake case e.g. entry_points
-# Boolean types (--minify)
-SUPPORTED_ESBUILD_APIS_BOOLEAN = [
-    "minify",
-    "sourcemap",
-]
-
-# single value types (--target=es2020)
-SUPPORTED_ESBUILD_APIS_SINGLE_VALUE = [
-    "target",
-    "format",
-    "main_fields",
-    "sources_content",
-]
-
-# Multi-value types (--external:axios --external:aws-sdk)
-SUPPORTED_ESBUILD_APIS_MULTI_VALUE = [
-    "external",
-    "loader",
-    "out_extension",
-]
+NON_CONFIGURABLE_VALUES = ["bundle", "platform", "outdir", "entry_points"]
```

Comment:
Are these the ones that is set by another field or by another parameter?

Reply:
These are ones the we set ourselves explicitly for it to work the way we went in Lambda.